zilliqa/src/p2p_node.rs

```diff
@@ -260,12 +259,6 @@ impl P2pNode {
 
                     let topic = Self::shard_id_to_topic(shard_id);
 
-                    // Push messages back into queue if there are no peers
```

Comment:
this interferes with broadcasts in consensus that you want to also recieve when you are the only node in the network. So, take it or leave it